Makefile

```diff
@@ -17,7 +17,7 @@ ifeq ($(OS),Darwin)
 endif
 
 ifeq ($(OS),Linux)
-	LD_LIBRARY_PATH += $(CURDIR)/operator/risc_zero/lib
+	export LD_LIBRARY_PATH+=$(CURDIR)/operator/risc_zero_old/lib:$(CURDIR)/operator/risc_zero/lib
```

Comment:
this should not have the `export` keyword

Reply:
Why not? Isn't the point to have this on the environment for child processes?

Reply:
we had a bug related to this previosly which was solved in https://github.com/yetanotherco/aligned_layer/pull/1299. Maybe the problem was another?